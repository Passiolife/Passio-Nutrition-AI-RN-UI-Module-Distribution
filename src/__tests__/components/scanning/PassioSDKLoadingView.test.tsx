// @ts-nocheck
import React from 'react';
import renderer from 'react-test-renderer';
import {
  BrandingProvider,
  ProgressLoadingView,
  ServicesProvider,
} from '@passiolife/nutrition-ai-ui-ux';
import { mockBranding, mockServices } from '../../provider/MockProviders';
describe('PassioSDKLoadingView', () => {
  it('renders correctly', async () => {
    jest.fn();
    const tree = renderer.create(
      <ServicesProvider services={mockServices}>
        <BrandingProvider branding={mockBranding}>
          <ProgressLoadingView />
        </BrandingProvider>
      </ServicesProvider>
    );
    expect(tree.toJSON()).toMatchSnapshot();
  });
});
