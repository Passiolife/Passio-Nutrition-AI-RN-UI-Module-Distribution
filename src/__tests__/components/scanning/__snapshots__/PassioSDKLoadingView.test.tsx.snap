// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PassioSDKLoadingView renders correctly 1`] = `
<View
  style={
    {
      "alignSelf": "center",
      "flex": 1,
      "flexDirection": "row",
      "justifyContent": "center",
    }
  }
>
  <ActivityIndicator
    accessibilityIgnoresInvertColors={false}
    color="#123456"
    size="large"
  />
  <Text
    style={
      {
        "alignSelf": "center",
        "color": "#4A4A4A",
        "fontSize": 18,
        "fontWeight": "600",
        "paddingHorizontal": 16,
      }
    }
  >
    Please Wait....
  </Text>
</View>
`;
